packages/builder/src/pages/builder/workspace/[application]/design/[workspaceAppId]/[screenId]/_components/AppPreview.svelte

```diff
@@ -272,7 +272,7 @@
   <iframe
     title="componentPreview"
     bind:this={iframe}
-    src="/app/preview"
+    src={`/app/${get(appStore).appId}/preview`}
```

Comment:
Would it not be simpler to just add a `/app/preview?appId=${$appStore.appId}`?

Also, you can just call `$appStore.appId` rather than `get(appStore)`